src/components/study/study-proposal-details.tsx

```diff
@@ -0,0 +1,111 @@
+import { use } from 'react'
+import { Divider, Grid, GridCol, Stack, Text, Badge } from '@mantine/core'
+import { AlertNotFound } from '@/components/errors'
+import { getMemberIdFromIdentifierAction } from '@/server/actions/user.actions'
+import { getStudyAction } from '@/server/actions/study.actions'
+import { dataForStudyDocumentsAction } from '@/server/actions/study-job.actions'
+import { Download } from '@phosphor-icons/react/dist/ssr'
+
+export function StudyProposalDetails(props: {
+    params: {
+        memberIdentifier: string
+        studyIdentifier: string
+    }
+}) {
+    const { memberIdentifier, studyIdentifier } = props.params
+
+    const member = use(getMemberIdFromIdentifierAction(memberIdentifier))
+
+    if (!member) {
+        return <AlertNotFound title="Member was not found" message="no such member exists" />
+    }
+
+    const study = use(getStudyAction(studyIdentifier))
+    if (!study) {
+        return <AlertNotFound title="Study was not found" message="no such study exists" />
+    }
+
+    const documents = use(dataForStudyDocumentsAction(studyIdentifier))
+    if (!documents) {
+        return <AlertNotFound title="Documents were not found" message="no such documents exists" />
+    }
+
```

Comment:
same comment as StudyCodeDetails, I think we can pass in member and study.  Maybe it would makes sense for this to check if the docs exists, but if we type the Study that's passed in to have those properties be non-optional then we wouldn't need this check as well